### PR TITLE
Add credential schema

### PR DESCRIPTION
**Pull request by nitro-neal**
Implement Credential Schema

A credential schema defines the structure and content of the data, enabling verifiers to assess if the data adheres to the established schema.

https://www.w3.org/TR/vc-data-model/#data-schemas

* Add CredentialSchema type
* Add optional field to VcCreateOptions
* Update KYC unit test to have latest kyc cred

### CODE REVIEW

File: credentials/src/test/kotlin/web5/sdk/credentials/VerifiableCredentialTest.kt [hunk 1]
```diff
@@ -167,25 +168,53 @@ class VerifiableCredentialTest {
...
+
+  val vc = VerifiableCredential.create(
+    type = "KnowYourCustomerCred",
+    issuer = issuerDid.uri,
+    subject = subjectDid.uri,
+    issuanceDate = Calendar.getInstance().time, // For the current time
+    expirationDate = expirationCalendar.time,
+    data = KnownCustomerCredential(id = subjectDid.uri, country_of_residence = "US", tier = "Tier 1"),
+    credentialSchema = CredentialSchema(
+      id = "https://schema.org/PFI",
```
Comment (decentralgabe) [inline credentials/src/test/kotlin/web5/sdk/credentials/VerifiableCredentialTest.kt:197]: we should use a real schema here and update the verification logic to resolve and verify the json schema against the credential

Comment (decentralgabe) [inline credentials/src/test/kotlin/web5/sdk/credentials/VerifiableCredentialTest.kt:197]: we should use a real schema here and update the verification logic to resolve and verify the json schema against the credential

**Review by decentralgabe — CHANGES_REQUESTED** [review]

File: credentials/src/test/kotlin/web5/sdk/credentials/VerifiableCredentialTest.kt (hunk 1 above)
Comment (nitro-neal) [reply credentials/src/test/kotlin/web5/sdk/credentials/VerifiableCredentialTest.kt:197]: Verification pr will be coming next 👍

**Review by decentralgabe — APPROVED** [review]
approving to unblock merge, noting verification will come in a separate pr

**Review by jiyoontbd — APPROVED** [review]

**Pull request merged**